src/status_im/chat/models/input.cljs

```diff
@@ -135,6 +135,24 @@
                                                :pack pack}
                                      :text    "Update to latest version to see a nice sticker here!"})))
 
+(fx/defn send-emoji-reaction-message
+  {:events [::send-emoji-reaction]}
+  [{{:keys [current-chat-id] :as db} :db :as cofx} {:keys [message-id emoji-id]}]
+  (chat.message/send-message cofx {:chat-id        current-chat-id
+                                   :content-type   constants/content-type-emoji-reaction
+                                   :emoji-reaction {:messageId message-id
+                                                    :emojiId   emoji-id}
+                                   :text           "Update to latest version to see a nice emoji reacion!"}))
+
+(fx/defn send-retract-emoji-reaction-message
+  {:events [::send-emoji-reaction-retraction]}
+  [{{:keys [current-chat-id] :as db} :db :as cofx} {:keys [message-id emoji-id]}]
+  (chat.message/send-message cofx {:chat-id        current-chat-id
+                                   :content-type   constants/content-type-emoji-reaction-retraction
+                                   :emoji-reaction {:messageId message-id
+                                                    :emojiId   emoji-id}
+                                   :text           "Update to latest version to see a nice emoji reacion!"}))
```

Comment:
This has the `content-type-emoji-reaction-retraction` content-type, but doesn't use the `emoji-reaction-retraction` object. It uses the `emoji-reaction` object